software/asp.net-core-inventory-order-management-system.yml

```
@@ -0,0 +1,12 @@
+name: "ASP.NET Core Inventory Order Management System"
+website_url: "https://github.com/go2ismail/Asp.Net-Core-Inventory-Order-Management-System"
+source_code_url: "https://github.com/go2ismail/Asp.Net-Core-Inventory-Order-Management-System"
+description: "Powerful and ready-to-use inventory management system, with a completely RESTful API for seamless integration and a modern front-end."
+licenses:
+  - CC-BY-SA-4.0
+platforms:
+  - .NET
+  - C#
+tags:
+  - Inventory Management
+demo_url: "https://github.com/go2ismail/Asp.Net-Core-Inventory-Order-Management-System?tab=readme-ov-file#live-demo-link"
```

```suggestion
demo_url: "https://github.com/go2ismail/Asp.Net-Core-Inventory-Order-Management-System?tab=readme-ov-file#live-demo-link"
depends_3rdparty: true
```

Requires IIS/Windows Server (https://github.com/go2ismail/Asp.Net-Core-Inventory-Order-Management-System?tab=readme-ov-file#run-the-project-iis-web-server)